<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>图像列表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>具体图像</string>
   </property>
  </widget>
  <widget class="QListWidget" name="ImgDirList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="ImgList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>241</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PreviewImg">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>图像预览</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="RecoButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>开始识别</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReImg1">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>识别出的人1</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ReImg2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>30</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>识别出的人2</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ReImg3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>260</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>识别出的人3</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ReImg4">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>260</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>识别出的人4</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ReName1">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名字1</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReName2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名字2</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReName3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>470</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名字3</string>
   </property>
  </widget>
  <widget class="QLabel" name="ReName4">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>470</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名字4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TrainButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始训练模型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FilteButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>502</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>过滤</string>
   </property>
  </widget>
  <widget class="QLabel" name="PreImgGrey">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>灰度图</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PreImgComPCA">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>PCA特征</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PreImgComLBP">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>LBP特征</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewAll">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>510</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查看所有特征图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testAllButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>全测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="shibielv">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0/0</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
